README.md

```diff
@@ -32,15 +33,23 @@ console.log(JSON.stringify(chunk.getBlock(new Vec3(3,50,3)),null,2))
 
 ## Test data
 
+### pc
+
 Test data can be generated with [minecraftChunkDumper](https://github.com/PrismarineJS/minecraft-chunk-dumper).
 
 Install it globally with `npm install minecraft-chunk-dumper -g` then run :
 
 `minecraftChunkDumper saveChunk 1.8.8 1.8/chunk.dump 1.8/chunk.meta`
 
-## API
+### bedrock
+
+Run tests in [bedrock-provider](https://github.com/PrismarineJS/bedrock-provider/) and copy data from `fixtures/$version/pchunk` into `tests/bedrock_$version`
```

Comment:
CCan

Reply:
The tests in bedrock-provider are done in combination with a client and server, since the server only loads and saves chunks after a player joins the server. The client tests there decode the chunks with the ChunkColumn implementation that was moved to pchunk, but doesn't do any special tests beyond that as to if network format re-encoding works or other chunk API methods.